src/cdi.rs

```diff
@@ -84,19 +83,20 @@ impl<N: ArrayLength<u8>, D: Digest, H: HmacImpl<D>> Zeroize for CompoundDeviceId
     }
 }
 
-impl<N: ArrayLength<u8>, D: Digest, H: HmacImpl<D>> ZeroizeOnDrop
+impl<const N: usize, D: Digest, H: HmacImpl<D>> ZeroizeOnDrop
     for CompoundDeviceIdentifier<N, D, H>
 {
 }
 
-impl<N: ArrayLength<u8>, D: Digest, H: HmacImpl<D>> CompoundDeviceIdentifier<N, D, H> {
+impl<const N: usize, D: Digest, H: HmacImpl<D>> CompoundDeviceIdentifier<N, D, H> {
     /// DICE CDI constructor.
     ///
     /// # Parameters
     /// @current_cdi: The CDI buffer.
     /// @cdi_type: The type of CDI
     pub fn new(cdi_bytes: &[u8], cdi_type: CdiType) -> Result<Self> {
-        let cdi = GenericArray::clone_from_slice(cdi_bytes);
+        //        let cdi = GenericArray::clone_from_slice(cdi_bytes);
```

Comment:
stray comment.

Reply:
Thanks. Fixed now, together with the subsequent `unwrap`.